### ISSUE TITLE
HTML and PDF versions

### ISSUE THREAD

**Issue by msuchard** [phylogeography/SpreaD3-tutorial#1]
Can pandoc render the LaTex file into a MarkDown file (i.e. README.md) as well for web display?  Or, should write the tutorial in MD and render in PDF for handouts and HTML for web display?

**Comment by fbielejec** [comment]
Good idea. @GuyBaele would that work for our webpage, linking to the md README hosted on github? Who can we task with this, @bramvrancken ?

**Comment by msuchard** [comment]
Or host at [Tutorial](http://phylogeography.github.io/SpreaD3-tutorial) ?   And maybe use something simple like [Bad example](https://github.com/barryclark/jekyll-now) to render MD?  Or [Super simple](http://kbroman.org/simple_site/)
